### ISSUE TITLE
[A11Y] - Make NetworkDropdown keyboard accessible

### ISSUE THREAD

**Issue by josephcoombes** [Uniswap/v3-info#215]
Currently cannot use keyboard navigation to access the `NetworkDropdown` menu.  This also means this element probably isn't accessible with screen-readers. To start, this could be as simple as converting the `div` to a `button` which is natively focusable. Alternatively, a `tab-index` attribute could be added to the `div`, but `button` is probably better solution.

Once the dropdown toggle is navigable by keyboard, the menu itself would also need to be keyboard accessible. Potentially the focus could/should be trapped in the dropdown once the user has keyboard navigated into it. Should be able to close the menu with `esc` key press. 

There are some different approaches highlighted here:

https://www.w3.org/WAI/tutorials/menus/flyout/#flyoutnavkbbtn
